src/main/java/ua/kpi/iasa/IASA_Organiser/controller/Controller.java

```diff
@@ -34,54 +49,128 @@ public void init(View view) {
         return eventService.getAllEvents();
     }
 
+    /**
+     * This method gets all events from the dataStorage.
+     * <p>
+     * Calls {@link EventService#filterExpiredEvents()} for removing all expired events,
+     * then calls {@link EventService#getAllEventsList()}.
+     *
+     * @return list of all events
+     * @see EventService
+     */
     public List<Event> getAllEventsList() {
         logger.debug("Method was called...");
         final EventService currentEventService = getEventService();
         currentEventService.filterExpiredEvents();
         return currentEventService.getAllEventsList();
     }
 
+    /**
+     * @deprecated It was useful at first lab, but now use {@link #getCalendarService()} instead.
+     */
     public Calendar getCalendar() {
         logger.debug("Method was called...");
         return calendarService.getCalendar();
     }
 
+    /**
+     * This method requests to save {@link Event} in argument into DB.
+     * <p>
+     * Receives Event instance, calls {@link EventService#createEvent(Event)} and puts received instance to it.
+     *
+     * @param event instance of Event
+     * @see EventService
+     */
     public void createNewEvent(Event event) {
         logger.debug("Method was called with {}", event);
         final EventService currentEventService = getEventService();
         currentEventService.createEvent(event);
     }
 
+    /**
+     * It request updated {@link Event} to restore it into DB .
+     * <p>
+     * Receives instance of Event, calls {@link EventService#updateEvent(Event)} and puts received instance to it.
+     *
+     * @param event Event that would be updated
+     * @see EventService
+     */
     public void updateEvent(Event event) {
         logger.debug("Method was called with {}", event);
         final EventService currentEventService = getEventService();
         currentEventService.updateEvent(event);
     }
 
+    /**
+     * This method executes {@link Event} deletion on DB.
+     * <p>
+     * Receives instance of Event, calls {@link EventService#removeEvent(Event)} and puts received instance to it.
+     *
+     * @param event
```

Comment:
Add little explanation what is event

Reply:
Have added explanation
Also to other event params